app/src/org/commcare/activities/EntityDetailActivity.java

```diff
@@ -224,6 +224,11 @@ protected boolean onBackwardSwipe() {
      * Move along to form entry.
      */
     private void select() {
+        // announce that case was selected
+        Intent selectIntentBroadcast = new Intent("org.commcare.dalvik.api.action.case.selected");

```

Comment:
could move this into a well-named helper and remove the need for an explanatory comment